vico/compose/src/main/java/com/patrykandpatrick/vico/compose/cartesian/CartesianChartHost.kt

```diff
@@ -146,7 +146,12 @@ internal fun CartesianChartHostImpl(
       ranges = ranges,
       scrollEnabled = scrollState.scrollEnabled,
       zoomEnabled = scrollState.scrollEnabled && zoomState.zoomEnabled,
-      layerPadding = chart.layerPadding,
+      layerPadding = remember(
+        chart.layerPadding,
+        model.extraStore,
+      ) {
+        chart.layerPadding(model.extraStore)
+      },
```

Comment:
There’s a formatting issue here. `ktfmt` makes this change:

```suggestion
      layerPadding =
        remember(chart.layerPadding, model.extraStore) { chart.layerPadding(model.extraStore) },
```

Reply:
I personally was struggling to run `ktfmt` here. At least in such a way that resulted in only a small change. I got the `ktfmt` plug-in, enabled it, and attempted to format - but noticed quite a lot changing.

I did adopt this change, though.

Reply:
`ktfmt` has multiple code styles. As mentioned in the contributing guidelines, we use the Google style. You probably forgot to set it. When using `ktfmt` directly (via the CLI), you apply it using `--google-style`. The plugin you used probably has a setting for the code style.